data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+				accept
+	on complete
+		log `Found some gullible Hai that pirates tricked into a "prison role-play cruise," complete with security guards and a brig. They overpowered the pirate crew who crashed the ship and used escape pods. I returned the Hai to <destination>.`
+		payment 565000
+		"hai slave prereq" ++
+		dialog `The Hai researchers thank you for rescuing them from their ill-fated "prison role-play cruise" and promise to stick to safer pastimes in the future. They happily give you <payment>.`
+	# No NPCs for this one since the Hai are presumed dead.
+
+
+mission "Hai Rescue: Ooonem"
+	landing
+	name "Return Ooonem"
+	description "Transport the Hai you rescued to <destination>. Payment is <payment>."
+	source "Stormhold"
+	destination "Makerplace"
+	passengers 1
+	repeat 3
+	to offer
+		"combat rating" > 50
+		"Hai Rescue: Ooonem (offer count)" < 3
+		not "event: hai-human treaty signed"
+		not "Hai Rescue: Ooonem: offered"
+		has "First Contact: Hai: offered"
+		has "Hai Leaks Response 6: active"
+		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.
+	on offer
+		"Hai Rescue: Ooonem (offer count)" ++
+		conversation
+			`In the shadows, you see a short, hooded, woman. There is a lot of hair sticking out from her hood. She might be a Hai. Do you want to talk to her?`
+			choice
+				`	(Talk to her.)`
+				`	(Come back later.)`
+					defer
+			`	As you approach, she pulls a laser rifle out of her cloak, points it at you, and says, "No! I'm not going back to them!"`
+			`	She sees your confusion and calms down, "You're not one of them. What do you want with me?"`
+			choice
+				`	"I was wondering why a Hai is on a pirate planet."`
+					goto hai
+				`	"Not one of whom?"`
+					goto who
+				`	"Sorry to bother you. I'll leave you to your business."`
+					goto decline
+			label hai
+			`	"I was going to meet a businessman on Shroud to arrange a large-scale import deal for my research project. I asked a human merchant to take me there, but I was tricked. The businessman doesn't really exist; the deal was faked. The merchant locked me in a cell and brought me here instead. When he opened the cell, I managed to grab his rifle, shoot him, and hide. It's amazing how slowly humans move."`
+				goto "more questions"
+			label who
+			`	"There are some men looking for me. I've managed to escape them so far; they're not as quick as a Hai. I think they're friends of the merchant who locked me up."`
+				goto "more questions"
+			label pay
+			`	"I have no money but my family on <planet> can pay."`
+				goto "more questions"
+			label "more questions"
+			`	Can you take me back to <destination>? My family can pay for my return.`
+			choice
+				`	"Yes, I'll take you there."`
+				`	"How much will you pay me?"`
+					goto pay
+				`	"Why are you on a pirate planet?"`
+					goto hai
+				`	"I'm not one of who?"`
```

Comment:
```suggestion
				`	"I'm not one of whom?"`
```

Reply:
And he questions me about the uncommon use of words in speech...

Reply:
Yeah, "whom" does seem uncommon, but "who" is actually grammatically incorrect here while "whom" is correct. [Grammarly on this subject](https://www.grammarly.com/blog/who-vs-whom-its-not-as-complicated-as-you-might-think/#:~:text=Whom%20should%20be%20used%20to,the%20subject%20of%20a%20sentence.).